chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -162,26 +162,26 @@ spec:
             periodSeconds: {{ .Values.scheduler.livenessProbe.periodSeconds }}
             exec:
               command:
-              - CONNECTION_CHECK_MAX_COUNT=0
-              - /entrypoint
-              - python
-              - -Wignore
-              - -c
-              - |
-                import os
-                os.environ['AIRFLOW__CORE__LOGGING_LEVEL'] = 'ERROR'
-                os.environ['AIRFLOW__LOGGING__LOGGING_LEVEL'] = 'ERROR'
+                - sh
+                - -c
+                - exec
+                - |
+                  CONNECTION_CHECK_MAX_COUNT=0 /entrypoint python -Wignore -c "
```

Comment:
Curious where `CONNECTION_CHECK_MAX_COUNT=0` is used

Reply:
Yeah, in entrypoint